api/src/pcapi/routes/native/v1/serialization/account.py

```diff
@@ -171,13 +171,19 @@ def get(self, key: str, default: typing.Any | None = None) -> typing.Any:
         if key == "subscriptionMessage":
             user_subscription_state = subscription_api.get_user_subscription_state(user)
             return user_subscription_state.subscription_message
+        if key == "activityId":
+            if user.activity is None:
+                return None
+            return profile_options.ActivityIdEnum(users_models.ActivityEnum(user.activity).name)
 
         return super().get(key, default)
 
 
 class UserProfileResponse(ConfiguredBaseModel):
+    activity_id: profile_options.ActivityIdEnum | None
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Question métier: on veut vraiment ce champ ? 
C'est un champ déclaratif, rempli à l'activation du crédit uniquement.. il est pas hyper fiable

Reply:
Oui, le ministère veut avoir une vue plus à jour de la classe sociale des bénéficiaires du pass Culture
On a un chantier pour [le mettre à jour](https://passculture.atlassian.net/browse/PC-30735) et [l'afficher dans les informations personnelles](https://passculture.atlassian.net/browse/PC-30734)